rskj-core/src/main/java/co/rsk/peg/federation/FederationMember.java

```diff
@@ -154,4 +159,14 @@ public int hashCode() {
                 mstPublicKey
         );
     }
+
+    public byte[] serialize() {
+        byte[][] rlpElements = new byte[FEDERATION_MEMBER_LIST_SIZE][];
+        rlpElements[FEDERATION_MEMBER_BTC_KEY_INDEX] = RLP.encodeElement(
+            this.getBtcPublicKey().getPubKeyPoint().getEncoded(true)
+        );
```

Comment:
```suggestion
        btcPublicKey = this.getBtcPublicKey().getPubKeyPoint().getEncoded(true);
        rlpElements[FEDERATION_MEMBER_BTC_KEY_INDEX] = RLP.encodeElement(
            btcPublicKey
        );
```
Same for the rest, to avoid having multiple function calls in a single line